### ISSUE TITLE
Release CU to support Cassandra driver version 4.x 

### ISSUE THREAD

**Issue by alivesubstance** [jsevellec/cassandra-unit#300]
I see master branch has already migrated to Cassandra driver 4.x version. Do you have a plans to release it and deploy into maven repository?

**Comment by bbottema** [comment]
Yeah, I'm also running into class not found exceptions, because cassandra unit is behind on the new driver.

**Comment by jsevellec** [comment]
I've planned to do it soon. There's already a snapshot available if you want to give it a try before the proper release

**Comment by sethishi** [comment]
Hi @jsevellec , would you be able to provide any sort of time frames for the release. I have been able to use the snapshot version fine, still testing it and no functional issues so far.

**Comment by jsevellec** [comment]
@sethishi thanks for the feedback. I'll release it soon

**Comment by sethishi** [comment]
Hi Guys, I see a new version on maven repository , assuming it is for the latest driver version 4.x
`
testCompile group: 'org.cassandraunit', name: 'cassandra-unit', version: '4.3.1.0'
`
My tests have started failing for the latest version compared to 4.2.2.0-SNAPSHOT. Are you aware of any compatibility issues?

**Comment by bbottema** [comment]
What's the current status?

**Comment by jsevellec** [comment]
The release has been done.  I missed this issue but yeah artifacts are available : 
https://github.com/jsevellec/cassandra-unit/wiki/changelog

**Comment by jsevellec** [comment]
**Cause.** The driver was still on the 3.x-era 4.3.1 pin.

**Fixed in #345.** Now `org.apache.cassandra:java-driver-core` 4.19.3 — the driver moved to Apache, and the DataStax coordinates are frozen at 4.17.0. Java packages are unchanged, so it is a pom change only.

Fixed on `main` and shipping in **5.0.0**, which is not on Maven Central yet — Central still has 4.3.1.0. Docs: https://github.com/jsevellec/cassandra-unit/tree/main/docs

Closing as fixed; sorry this took so long.
